webapp-django/crashstats/settings/base.py

```diff
@@ -567,6 +567,12 @@ def filter(self, record):
     implementations_database_url = database_url
 implementations_config = dj_database_url.parse(implementations_database_url)
 
+# The CrashQueueBase class to use for submitting priority and reprocessing
+# requests
+CRASHQUEUE = config(
+    "queue.crashqueue_class", "socorro.external.pubsub.crashqueue.PubSubCrashQueue"
+)
+
```

Comment:
This continues the trend of having the webapp use processor environment configuration variables. We can't do it the other way around--configman doesn't work like that. Not having duplicated values makes life easier.

At some point in the future, we'll switch from configman to something else and I'm hoping some of these problems will go away then. We'll see.